components/controls/RadioCard.css

```diff
@@ -66,6 +71,15 @@
   border-color: var(--darkBlue);
 }
 
+.radio-card-vertical:not(.disabled):hover {
```

Comment:
Shouldn't it be `.radio-card-actionable.radio-card-vertical....` ? We never want the hover effect if the card is not actionable.